src/cljs/netrunner/gameboard.cljs

```diff
@@ -1278,6 +1278,67 @@
   (when sfx-current-id
     (om/set-state! owner :sfx-last-played {:gameid gameid :id sfx-current-id})))
 
+(def corp-stats
+  (let [s #(-> @game-state :stats :corp)]
+    [["Clicks Gained" #(-> (s) :gain :click)]
+     ["Credits Gained" #(-> (s) :gain :credit)]
+     ["Credits Lost" #(-> (s) :lose :credit)]
+     ["Credits by Click" #(-> (s) :click :credit)]
+     ["Cards Drawn" #(-> (s) :gain :card)]
+     ["Cards Drawn by Click" #(-> (s) :click :draw)]]))
+
+(def runner-stats
+  (let [s #(-> @game-state :stats :runner)]
+    [["Clicks Gained" #(-> (s) :gain :click)]
+     ["Credits Gained" #(-> (s) :gain :credit)]
+     ["Credits Lost" #(-> (s) :lose :credit)]
+     ["Credits by Click" #(-> (s) :click :credit)]
+     ["Cards Drawn" #(-> (s) :gain :card)]
+     ["Cards Drawn by Click" #(-> (s) :click :draw)]
+     ["Tags Gained" #(-> (s) :gain :tag)]
+     ["Runs Made" #(-> (s) :runs :started)]
+     ["Cards Accessed" #(-> (s) :access :cards)]]))
+
+(defn show-stat
+  "Determines statistic counter and if it should be shown"
+  [side]
+  (when-let [stat-fn (-> side second)]
+    (let [stat (stat-fn)]
+      (if (pos? stat) stat "-"))))
+
+(defn build-game-stats
+  "Builds the end of game statistics div & table"
+  []
+  (let [stats (map-longest list nil corp-stats runner-stats)]
+    [:div
+     [:table.win.table
+      [:tr.win.th
+       [:td.win.th "Corp"] [:td.win.th]
+       [:td.win.th "Runner"] [:td.win.th]]
+      (for [[corp runner] stats]
+        [:tr [:td (first corp)] [:td (show-stat corp)]
+         [:td (first runner)] [:td (show-stat runner)]])]]))
+
+(defn build-win-box
+  "Builds the end of game pop up game end"
+  [game-state]
+  [:div.win.centered.blue-shade
+   [:div
+    (:winning-user @game-state) " (" (-> @game-state :winner capitalize)
+    (cond
+      (= "Decked" (@game-state :reason capitalize))
+      (str ") wins due to the Corp being decked on turn " (:turn @game-state))
+
+      (= "Flatline" (@game-state :reason capitalize))
+      (str ") wins by flatlining the Runner on turn " (:turn @game-state))
+
+      :else
+      (str ") wins by scoring agenda points on turn "  (:turn @game-state)))]
```

Comment:
I'd like an explicit "win by concession" as well for when the opponent uses the concede button.